vmdb/app/models/ems_refresh/save_inventory_orchestration_stacks.rb

```diff
@@ -15,11 +15,18 @@ def save_orchestration_templates_inventory(_ems, hashes, _target = nil)
       hashes.zip(templates).each { |hash, template| hash[:id] = template.id }
     end
 
+    def all_orchestration_stacks(ems)
+      if ems.respond_to?(:orchestration_stacks_with_nested)
+        ems.orchestration_stacks_with_nested(true)
+      else
+        ems.orchestration_stacks(true)
+      end
+    end
+
```

Comment:
I'll try to check it out as soon as I can, this task has very low priority now.

You did this change and tests are passing?

I changed it this way so list of stacks in the UI shows only top level stacks, not all of them, it can be confusing.